### PR TITLE
allow the ember divshot command to take divshot option arguments

### PR DESCRIPTION
**Pull request by klclee**

### CODE REVIEW

**Comment by stefanpenner** [comment]
nice!

File: tests/unit/commands/divshot-nodetest.js [hunk 1]
```diff
@@ -59,6 +59,20 @@ describe('divshot command', function() {
     }).validateAndRun(['push']);
   });
 
+  it('accecpt `divshot` option arguments', function() {
```
Comment (Dhaulagiri) [inline tests/unit/commands/divshot-nodetest.js]: minor typo: accecpt -> accepts

**Comment by rwjblue** [comment]
Looks like this makes `ember divshot push token 1234` work, can we make `ember divshot push --token 1234` work instead?

**Comment by klclee** [comment]
thx @Dhaulagiri well spotted :+1:  @rwjblue i tried that but seems like ember cli sucks up all the '--<arg>' so the ```run``` function never see those. Is there away for ember cli to pass that on? If so agree is defo a better experience!

**Comment by rwjblue** [comment]
IIRC we should be able to specify `availableOptions` which will be used for parsing.  See https://github.com/ember-cli/ember-cli/blob/master/lib/commands/test.js#L15-L23 for details.  Support for additional available options from custom commands was added in https://github.com/ember-cli/ember-cli/pull/2565.

**Comment by klclee** [comment]
@rwjblue thanks for the tip! it now allows '--args'.

**Comment by rwjblue** [comment]
Thanks!! Released as 0.1.7.

**Pull request merged**